pkg/configuration/parser.go

```diff
@@ -118,21 +116,6 @@ func (p *nsxConfigParser) storeParsedSegments() {
 	}
 }
 
-func (p *nsxConfigParser) removeVMsWithoutGroups() {
```

Comment:
I would keep this function, and add a warning if there are such VMs, that they are ignored.

Reply:
done